kubernetes/helm/pinot/values.yaml

```diff
@@ -305,7 +311,10 @@ server:
     nodePort: ""
     protocol: TCP
 
-  resources: {}
+  resources:
+    requests:
+      cpu: "2000m"
+      memory: "1.25Gi"
```

Comment:
should we set different value for server/minion from broker/controller. the usage pattern is drastically different right?

Reply:
I understand that they are quite different, but these resource requests have been decided taking into account the Xmx and GC that is being used. As all broker, controllers and servers use the same Xmx (1G) and the same GC (G1) then all of them have the same requests.

I may make sense to change the default JAVA_OPTS for each service, but I think that is out of the scope of this PR

Reply:
sounds good